.github/workflows/CI.yml

```diff
@@ -240,41 +240,38 @@ jobs:
           export TEL_KEY=`node -e "console.log(Buffer.from('${{ secrets.TELEMETRY_KEY }}').toString('base64'))"`
           sed -i "s/const TELEMETRY_KEY_ENCODED = TELEMETRY_DEFAULT_KEY/const TELEMETRY_KEY_ENCODED = '$TEL_KEY'/" telemetry.ts
       - name: Package VSIX
-        run: cd clients/vscode-hlasmplugin/ && npm ci && npm run compile && npm run package
-      - name: Get VSIX filename
-        run: echo VSIX_FILENAME=$(ls clients/vscode-hlasmplugin/*.vsix) >> $GITHUB_ENV
-      - name: Get version
-        run: echo VERSION=$(node -e "console.log(require('./clients/vscode-hlasmplugin/package.json').version)") >> $GITHUB_ENV
+        run: cd clients/vscode-hlasmplugin/ && npm ci && npm run compile && npx vsce package -o hlasm-language-support.vsix --baseContentUrl "https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/"
       - name: Upload VSIX
         uses: actions/upload-artifact@v3
         with:
-          name: hlasm-language-support-${{ env.VERSION }}.vsix
-          path: ${{ env.VSIX_FILENAME }}
+          name: hlasm-language-support.vsix
+          path: clients/vscode-hlasmplugin/hlasm-language-support.vsix
```

Comment:
Just a suggestion - maybe we can have a variable for `clients/vscode-hlasmplugin`

Reply:
There seem to be other opportunities on improving the workflow definition as well... So, I will look into that in another PR.